articles/openshift/oshift-ref-dns-zone-forwarding.md

```diff
@@ -0,0 +1,68 @@
+This article is best aimed at those with OpenShift deployments with connectivity to a VRF as other networks commonly have open resolvers available.
+
+## OpenShift Cluster DNS Architecture
+
+We deploy HAProxy load balancer pairs as part of an OpenShift deployment; primarily to provide redundancy for inbound traffic. As a minimum, all clusters will have: 
+
+- control plane load balancers forwarding to routes:
+  - OpenShift API/Web Console
+  - Docker Registry Console
+  - Grafana
+  - Hawkular Metrics
+  - Kibana
+  - Prometheus
+
+- data plane load balancers forwarding to routes exposed on the default OpenShift routers
+
+Deployments with connectivity to government community networks have an additional data plane load balancer pair (forwarding to routes exposed on the secondary OpenShift routers) which only have inbound connectivity from the secondary network.
+
+On the control plane load balancers, we configure `BIND` (also referred to as `named`) to resolve the local zone within the cluster (performing internal hostname resolution) and all other zones are forwarded outbound to an open internet resolver. All internet OpenShift nodes reference the control plane load balancers as resolvers.
+
+Where a deployment has connectivity with government community networks, we forward the local zone and a small number of internet zones (required for provisioning and ongoing maintenance) to the control plane BIND service, with all remaining zones being forwarded to open resolvers on the secondary network. All secondary network nodes reference the secondary network data plane load balancers as resolvers.
+
+## Identifying Forward Zones
+
+A zone is a domain (for example domain.com) and configuring a forward zone will forward any DNS queries received for that domain and all sub-domains to the configured IP address. This allows different domains to be resolved by separate DNS servers at a granular level if required.
+
+In OpenShift deployments with connectivity to a VRF, you may want to resolve a pre-existing internal zone using a private resolver, instead of forwarding these requests to an external service (and therefore routing traffic over an external network) which gives you the benefit of being able to locate private services that may otherwise be restricted or inaccessible from these networks. Such examples of services that could be consumed from OpenShift include: container image registry, binary repository, version control system.
+
+## Requesting DNS Zone Forwarding
+
+When provisioning your environment (and provided you have highlighted that you require connectivity with a VRF) we'll contact you to ask if there are any zones which you want to forward and request the IP addresses of the corresponding DNS resolvers. To add forward zones, we add the provided details to the `named.conf` file on the control plane load balancers (and the secondary network load balancers if they exist) and restart the `named` service on these nodes to apply the changes.
```

Comment:
Again, remove VRF. Also we don't contact to ask, they can specify it as part of cluster deployment or we can do it through a service request.
We don't need the technical details of the config files etc here. They'll never have visibility of that.